client/lcd/lcd_test.go

```diff
@@ -1017,16 +1021,31 @@ func getDelegatorUnbondingDelegations(t *testing.T, port string, delegatorAddr s
 	return ubds
 }
 
-func getDelegatorRedelegations(t *testing.T, port string, delegatorAddr sdk.AccAddress) []stake.Redelegation {
-	res, body := Request(t, port, "GET", fmt.Sprintf("/stake/delegators/%s/redelegations", delegatorAddr), nil)
-	require.Equal(t, http.StatusOK, res.StatusCode, body)
+func getRedelegations(t *testing.T, port string, delegatorAddr sdk.AccAddress, srcValidatorAddr sdk.ValAddress, dstValidatorAddr sdk.ValAddress) []stake.Redelegation {
+	var res *http.Response
+	var body string
+
+	endpoint := "/stake/redelegations?"
+
+	if !delegatorAddr.Empty() {
+		endpoint += fmt.Sprintf("delegator=%s&", delegatorAddr)
```

Comment:
consider using https://golang.org/pkg/strings/#Builder

Reply:
hmm interesting - yeah seems like it would make the code a bit smaller - but I don't see builders often so might just be more confusing for new devs... but then again maybe we should all learn!

Reply:
I think in this situation, it is simple enough that we can just use the `+=` for the test.  But I agree, in more important places, where we want to reduce memory copying, a String builder is good to use.